playwright.config.ts

```diff
@@ -0,0 +1,115 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './src/tests/playwright',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: 'http://127.0.0.1:8080/src/tests/playwright/',
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'on-first-retry',
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: 'chromium',
+      use: { 
+        ...devices['Desktop Chrome'], 
+        launchOptions: { 
+          ignoreDefaultArgs: ['--mute-audio'],
+          args: ['--autoplay-policy=no-user-gesture-required']
+        },
+      }
+    },
+
+    {
+      name: 'chromium custom',
+      use: {
+        ...devices['Desktop Chrome'], 
+        launchOptions: {
+          executablePath: './chromium-browsers/chrome-123-0-6296-0/Google Chrome for Testing.app/Contents/MacOS/Google Chrome for Testing',
+          ignoreDefaultArgs: ['--mute-audio'],
+          args: ['--autoplay-policy=no-user-gesture-required']
+        }
+      }
+    }
+
+    // {
```

Comment:
For landing this PR, we want to delete anything that is not used.

But I do appreciate seeing these options are available! :)